go-controller/pkg/libovsdb/util/acl.go

```diff
@@ -88,10 +88,10 @@ func GetACLName(dbIDs *libovsdbops.DbObjectIDs) string {
 	return fmt.Sprintf("%.63s", aclName)
```

Comment:
could we get the isolation changes in one commit and the renaming in another? I assume some of this is functional (isolation bits) and some of this is renaming/cleaningup? -> or maybe its hard to split it?